packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security.cy.ts

```diff
@@ -0,0 +1,59 @@
+describe("TESTS: Access secured by Elytron OIDC client", () => {
+  let wildfly: string;
+  let ssoServer: string;
+  const profileInfo =
+    "#hal-header-container > div.collapse.navbar-collapse.hal-header-collapse > ul.nav.navbar-nav.navbar-utility > li:nth-child(5) > a";
+
+  before(() => {
+    cy.startWildflyContainerSecured()
+      .then((result) => {
+        wildfly = result as string;
+      })
+      .then(() => {
+        cy.startKeycloakContainer().then((result) => {
+          ssoServer = result as string;
+          // the following CLI commands setup the OIDC configuration
+          cy.executeInWildflyContainer(
+            `"/subsystem=elytron-oidc-client/provider=keycloak:add(provider-url=${ssoServer}/realms/wildfly-infra)"`
+          )
+            .then(() => {
+              return cy.executeInWildflyContainer(
+                `"/subsystem=elytron-oidc-client/secure-deployment=wildfly-management:add(provider=keycloak,client-id=wildfly-management,principal-attribute=preferred_username,bearer-only=true,ssl-required=EXTERNAL)"`
+              );
+            })
+            .then(() => {
+              return cy.executeInWildflyContainer(
+                `"/subsystem=elytron-oidc-client/secure-server=wildfly-console:add(provider=keycloak,client-id=wildfly-console,public-client=true)"`
+              );
+            })
+            .then(() => {
+              cy.executeInWildflyContainer(`"reload"`);
+            });
+        });
+      });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Logs in successfully and logs out", () => {
+    cy.visit(`?connect=${wildfly}#home`);
+    cy.get("#username").type("userwithoutrole");
+    cy.get("#password").type("password");
+    cy.get("#kc-login").click();
+    cy.url().should(`include`, `localhost:${Cypress.env("HAL_CONTAINER_PORT") as string}`);
+    cy.get(profileInfo).click().get("#logout-link").click();
```

Comment:
> cy.get(profileInfo).click().get("#logout-link").click();

I suggest to create a logout helper function. It can be used at least in these two new test specs.